### DISCUSSION TITLE
Dropdown display for grouped entry types

### DISCUSSION THREAD

**Discussion by samuelreichor** [craftcms/cms#13184] [Ideas] [answered]
For particularly large craft projects with a lot of sections, we find that the sidebar can quickly become very confusing. The feature with the headlines is very cool, but it could be even better. We would like it to be possible to group sections and then display them as a dropdown in the sidebar. Maybe an idea for Craft 5? 
![image](https://github.com/craftcms/cms/assets/98765339/38817215-2429-45b8-861d-c3329bec6fea)

**Comment by brandonkelly** [comment]
As in, a second dropdown in addition to the site selector?

**Comment by samuelreichor** [comment]
Something like that:

![image](https://github.com/craftcms/cms/assets/98765339/c4528148-a2f6-4446-9451-39aa441bf460)

**Comment by brandonkelly** [comment, marked as answer]
Thanks! This was previously requested by #11340, so going to close this as a duplicate.

**Reply by samuelreichor** [reply]
Ah sorry thanks :)
